### ISSUE TITLE
Function to auto download cycleways

### ISSUE THREAD

**Issue by Robinlovelace** [saferactive/traffiCalmr#15]
highway=path + bicycle=designated , highway=path + cycleway=lane , highway=* + cycleway=track , and highway=cycleway

https://twitter.com/rjw62/status/1273241796110356480

**Comment by PublicHealthDataGeek** [comment]
should this be a function to specifically download OSM cycleways?  The CID also has cycleways...  do we want that in too? NB strengths and limitations of both datasets

**Comment by Robinlovelace** [comment]
OSM data in the 1st instance but CID data also.

**Comment by layik** [comment]
I take it this would need to be done by some "place" name or do we mean nationally? I can see an area but cannot see the use of a national use immediately in a package.

**Comment by layik** [comment]
CID = Cycling Infrastructure Database would then be consolidating both "place == London" & nationally.

**Comment by Robinlovelace** [comment]
> I take it this would need to be done by some "place" name or do we mean nationally?

Agreed, like the traffic calming function it could start with OSM data from a particular place. No harm in being able to get it nationally though. I think the CID-getting functionality may be best as a separate function as it's quite specific to London and less generalisable nationally and internationally.

**Comment by layik** [comment]
Ageeed. Thanks Robin and CT!

**Comment by PublicHealthDataGeek** [comment]
Agreed - a new and separate function for CID derived data on cycleways.  Do I need to raise that as a new issue? 
Am happy to develop it but may need some assistance in terms of adding to package etc.

**Comment by Robinlovelace** [comment]
> Do I need to raise that as a new issue?

Good idea. If you're up for 'fixing' it also (can guide you on that Friday or next week), even better!
